storage/s3/templates.go

```diff
@@ -0,0 +1,31 @@
+package s3
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+const bucket = "BUCKET HERE"
```

Comment:
os.Getenv("TEMPLATES_BUCKET")

Reply:
This whole file is removed with latest changes with the service.